src/plugins/mdraid.c

```diff
@@ -357,8 +357,7 @@ static BDMDExamineData* get_examine_data_from_table (GHashTable *table, gboolean
         }
 
         if (bs_error) {
-            g_set_error (error, BD_MD_ERROR, BD_MD_ERROR_PARSE,
-                         "Failed to parse chunk size from mdexamine data: %s", bs_error->msg);
+            g_warning ("get_examine_data_from_table(): Failed to parse chunk size from mdexamine data: %s", bs_error->msg);
```

Comment:
Why this change?

Reply:
Ah, that. We've agreed with @vojtechtrefny that this shouldn't fail the call - it's a single attribute that is not that important when missing. This change also mimics the previous behaviour where data have been returned successfully and the error got simply lost.

Reply:
OK